Patch taken from upstream cvs to fix funky blue tinting.

--- src/pixmap.c
+++ src/pixmap.c
@@ -54,2 +54,7 @@
 
+/* FIXME:  Workaround for older versions of libast. */
+#ifndef WORDS_BIGENDIAN
+#  define WORDS_BIGENDIAN 0
+#endif
+
 /* Assembler routines */
@@ -1636,3 +1641,3 @@
 
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 r = (ptr[x + 1] * rm) >> 8;
@@ -1659,3 +1664,3 @@
 
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 r = (ptr[x + 1] * rm) >> 8;
@@ -1671,3 +1676,3 @@
                 b |= (!(b >> 8) - 1);
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 ptr[x + 1] = r;
@@ -1701,3 +1706,3 @@
 
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 r = (ptr[x + 0] * rm) >> 8;
@@ -1724,3 +1729,3 @@
 
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 r = (ptr[x + 0] * rm) >> 8;
@@ -1736,3 +1741,3 @@
                 b |= (!(b >> 8) - 1);
-# ifdef WORDS_BIGENDIAN
+# if WORDS_BIGENDIAN
                 ptr[x + 0] = r;
